packages/ListBox/src/components/Trigger/Trigger.js

```diff
@@ -1,100 +1,84 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { v4 as uuidv4 } from "uuid";
 import RawButton from "@paprika/raw-button";
 import Button from "@paprika/button";
-import CaretDownIcon from "@paprika/icon/lib/CaretDown";
-import CaretUpIcon from "@paprika/icon/lib/CaretUp";
-import TimesCircleIcon from "@paprika/icon/lib/TimesCircle";
 import useI18n from "@paprika/l10n/lib/useI18n";
 import Label from "../Label";
 import { handleKeyDownKeyboardKeys, handleKeyUpKeyboardKeys } from "../../helpers/handleKeyboardKeys";
 import useListBox from "../../useListBox";
 import { OnChangeContext } from "../../store/OnChangeProvider";
-
+import { PropsContext } from "../../store/PropsProvider";
 import invokeOnChange, {
   sanitizeActionTypes,
   getSelectedOptionSingle,
   getSelectedOptionsMulti,
 } from "../../helpers/invokeOnChange";
-
 import * as sc from "./Trigger.styles";
 import { getDOMAttributesForListBoxButton } from "../../helpers/DOMAttributes";
 
-const propTypes = {
-  /** Custom clear icon */
-  clearIcon: PropTypes.node,
-
-  /** Body content of the trigger. */
-  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
-
-  /** If true it adds a clear button */
-  hasClearButton: PropTypes.bool,
-
-  /** Has implicit "All items selected" value when no item is selected */
-  hasImplicitAll: PropTypes.bool,
-
-  /** Override the label with a custom one. */
-  label: PropTypes.string,
-
-  /** Callback to be executed when the clear button is clicked or activated by keyboard. */
-  onClickClear: PropTypes.func,
-
-  /** Callback to be executed when the accept button is clicked or activated by keyboard. */
-  onClickFooterAccept: PropTypes.func,
-
-  /** Sets a placeholder for the trigger */
-  placeholder: PropTypes.string,
-
-  /** If true the trigger will be hidden */
-  isHidden: PropTypes.bool,
-};
-
-const defaultProps = {
-  clearIcon: null,
-  children: <React.Fragment />,
-  hasClearButton: true,
-  hasImplicitAll: false,
-  label: null,
-  onClickClear: null,
-  onClickFooterAccept: null,
-  placeholder: "Select...",
-  isHidden: false,
-};
-
 export default function Trigger(props) {
+  const I18n = useI18n();
   const [state, dispatch] = useListBox();
   const onChangeContext = React.useContext(OnChangeContext);
+  const providedProps = React.useContext(PropsContext);
+
   const {
+    children,
     clearIcon,
-    placeholder,
     hasClearButton,
     hasImplicitAll,
-    label,
-    onClickFooterAccept,
-    children,
     isHidden,
+    label,
     onClickClear,
-    // eslint-disable-next-line react/prop-types
-    onFooterClickAccept,
```

Comment:
This wasn't used, except [here](https://github.com/acl-services/paprika/pull/954/files#diff-ed9bd4d6b0e06d0607863760d0a10b00f3db83598b7d52e31fbacf3a801f7b20L45) and that seemed like a bug.  `onClickFooterAccept` is used instead.